lib/octicons_styled/script/build.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+const path = require('path')
+const fs = require('fs-extra')
+const octicons = require('../../octicons_react')
+
+const GENERATED_HEADER = '/* THIS FILE IS GENERATED. DO NOT EDIT IT. */'
+
+const generatedDir = path.join(__dirname, '../src/__generated__')
+const iconsDir = path.join(generatedDir, 'icons')
+
+fs.ensureDirSync(iconsDir)
+
+const icons = Object.keys(octicons).filter(name => name !== 'default')
+
+const initialTypeDefinitions = `${GENERATED_HEADER}
+import * as React from 'react'
+import * as StyledSystem from 'styled-system'
+import {SystemStyleObject} from '@styled-system/css'
+
+type Size = 'small' | 'medium' | 'large'
+
+interface IconProps extends StyledSystem.ColorProps, StyledSystem.SpaceProps {
+  'aria-label'?: string
+  className?: string
+  size?: number | Size
+  verticalAlign?: 'middle' | 'text-bottom' | 'text-top' | 'top' | 'unset'
+  sx?: SystemStyleObject
+}
+
+type Icon = React.FC<IconProps>
```

Comment:
Would it be helpful if exported these types? @dmarcey

Reply:
Yes! I think it would be nice to export both `IconProps` and `Icon`.